puppet/modules/chassis/manifests/wp.pp

```diff
@@ -56,4 +56,8 @@
 	file { '/vagrant/local-config-extensions.php':
 		content => template('chassis/local-config-extensions.php.erb')
 	}
+
+	file { '/home/vagrant/.wp-cli/config.yml':
+		content => template('chassis/wp-cli.yml')
```

Comment:
@shadyvb Could you please change this to be `chassis/wp-cli.yml.erb` please and also rename the the file accordingly as well? Thank you! ❤️